scripts/api-inference/scripts/generate.ts

```diff
@@ -446,35 +450,54 @@ TASKS.forEach((task) => {
 ///////////////////////////////////////////////
 
 function fetchChatCompletion() {
-  // Recommended models based on text-generation
-  DATA.models["chat-completion"] = DATA.models["text-generation"].filter(
-    // @ts-ignore
-    (model) => model.config?.tokenizer_config?.chat_template,
-  );
+  const conversationalTasks = [
+    {
+      name: "conversational-text-generation",
```

Comment:
I think the link issue from comment above comes from this `name`. If fixing this starts to be too difficult, I don't mind if you hardcode the template for both subtasks.

Reply:
indeed! addressed in this commit: [05b8858](https://github.com/huggingface/hub-docs/pull/1440/commits/05b88586319f402feffc2b567067489b0533ae82)